app/components/A/tests/index.test.js

```diff
@@ -22,7 +22,7 @@ describe('<A />', () => {
     expect(renderedComponent.type()).toEqual('a');
   });
 
-  it('should have an href attribute', () => {
+  it('should have a href attribute', () => {
```

Comment:
Pretty much all other tests regarding attributes in this codebase are worded in this way *(e.g. 'should have a ? attribute')* and this is why I changed this one to match. IMO **a** also reads better than **an** in this instance. I can change it back if you think it should be **an**, but does that mean all the others should also be updated?